main.go

```diff
@@ -209,7 +210,18 @@ func Flash(pkgName, port string, options *compileopts.Options) error {
 			flashCmd = strings.Replace(flashCmd, "{port}", port, -1)
 
 			// Execute the command.
-			cmd := exec.Command("/bin/sh", "-c", flashCmd)
+			var cmd *exec.Cmd
+			switch runtime.GOOS {
+			case "windows":
+				command := strings.Split(flashCmd, " ")
```

Comment:
It's better to split commands using a dedicated command splitter, such as github.com/google/shlex, otherwise spaces may give invalid results.

Reply:
Was not aware of that package, just switched to using it.

Reply:
Hmm shlex does not appear to work as I expected, and now my `wmic` calls are failing. Have to look deeper into how it works, but my previous solution was working well enough.

Basically just separating the command from the params is probably enough, since what was happening was the runtime on Windows does some extra insertions of `\` characters.

Anyhow, I think I will keep the other changes, but go back to my simpler parsing that works for the Windows rules.

Reply:
Just a little bit worried that future changes may trip something up, but I guess it's fine for now.